src/client/components/pages/revision.js

```diff
@@ -105,9 +112,29 @@ class RevisionPage extends React.Component {
 		return title;
 	}
 
+	constructor(props) {
+		super(props);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	handleSubmit(event) {
+		event.preventDefault();
+		const data = {
+			note: this.noteInput.getValue()
+		};
+		console.log(data);
+		request.post(`/revision/${this.props.revision.id}/note`)
+			.send(data).promise()
+			.then(() => {
+				location.reload();
```

Comment:
I'm pretty sure the other components have reloads. This preferably should be done with an XHR though since the user would be guaranteed to have JS enabled if they made a comment. Consider making this a task on GCI? :)